tests/Integration/Schema/Directives/AllDirectiveTest.php

```diff
@@ -200,8 +201,54 @@ public function testSpecifyCustomBuilder(): void
         ]);
     }
 
+    public function testSpecifyCustomBuilderForRelation(): void
+    {
+        $post = factory(Post::class)->create();
+
+        $this->schema = /** @lang GraphQL */ '
+        type Post {
+            id: ID!
+        }
+
+        type User {
+            id: ID!
+            posts: [Post!]! @all(builder: "' . $this->qualifyTestResolver('builderForRelation') . '")
+        }
+
+        type Query {
+            user(id: ID!): User @find
+        }
+        ';
+
+        // The custom builder is supposed to change the sort order
```

Comment:
Same as the other test, it would be good if this test were to prove that the builder was actually used.